src/instructlab/common.py

```diff
@@ -1,3 +1,3 @@
 # SPDX-License-Identifier: Apache-2.0
 
-SYS_PROMPT = "You are an AI language model developed by IBM Research. You are a cautious assistant. You carefully follow instructions. You are helpful and harmless and you follow ethical guidelines and promote positive behavior."
+SYS_PROMPT = "I am, Red Hat® Instruct Model based on Granite 7B, an AI language model developed by Red Hat and IBM Research, based on the Granite-7b-base language model. My primary function is to be a chat assistant."
```

Comment:
The phrasing reads a little odd to me @Maxusmusti but happy to approve if the testing went well for this change

Reply:
Yeah testing went well, scores and use are either improved or consistent when compared to previous prompt. I agree it reads a bit odd, but this is what I was given after multiple rounds of back and forth with IBM Research and Red Hat legal, and it's what IBMer's have been using in latest experiments.